src/build.sh

```diff
@@ -365,6 +373,15 @@ function parse_build_options()
         options_values['FULL_CLEANUP']=1
         shift
         ;;
+      --cflags)
+        if [[ -z "$2" ]]; then
+          options_values['CFLAGS']="${build_config[cflags]}"
+          shift
+        else
```

Comment:
At the beginning of the `parse_build_options` you already set CFLAGS, I guess you don't need this part of the code.

Reply:
Thx. I previously made this modification but forgot to remove it.